docs/source/quantization.rst

```diff
@@ -132,12 +141,17 @@ Operations that are available from the ``torch`` namespace or as methods on Tens
 * :meth:`~torch.Tensor.q_per_channel_scales` — Returns the scales of the per-channel quantized tensor
 * :meth:`~torch.Tensor.q_per_channel_zero_points` — Returns the zero points of the per-channel quantized tensor
 * :meth:`~torch.Tensor.q_per_channel_axis` — Returns the channel axis of the per-channel quantized tensor
-* :meth:`~torch.Tensor.relu` — Rectified linear unit (copy)
-* :meth:`~torch.Tensor.relu_` — Rectified linear unit (inplace)
 * :meth:`~torch.Tensor.resize_` — In-place resize
 * :meth:`~torch.Tensor.sort` — Sorts the tensor
 * :meth:`~torch.Tensor.topk` — Returns k largest values of a tensor
 
+``torch.nn.functional``
+~~~~~~~~~~~~~~~~~~~~~~
+
+Basic activations are supported.
```

Comment:
To avoid confusion,  we should also add 

.. autofunction:: max_pool2d
.. autofunction:: adaptive_avg_pool2d
.. autofunction:: avg_pool2d
.. autofunction:: interpolate
.. autofunction:: upsample
.. autofunction:: upsample_bilinear
.. autofunction:: upsample_nearest

We do support calling these functions as either nn.functional and nn.quantized.functional

Reply:
added